docs/5___ssd.adoc

```diff
@@ -144,6 +144,27 @@ Boolean / String / Enumeration / Binary |Exactly one of these elements *CAN* be
 
 The type of the Connector is identified by the presence of one of the XML child elements Real, Float64, Float32, Integer, Int8, UInt8, Int16, UInt16, Int32, UInt32, Int64, UInt64, Boolean, String, Enumeration, or Binary.
 
+When Modelica models are represeented in SSP, built-in input and output connectors shall be mapped as follows.
+
+[width="100%",cols="25%,25%,50%",options="header",]
+|===
+|Modelica Type |SSP Type |SSP Kind
+|RealInput |ssc:Real |input
+|RealOutput |ssc:Real |output
+|IntegerInput |ssc:Integer |input
+|IntegerOutput |ssc:Integer |output
+|BooleanInput |ssc:Boolean |input
+|BooleanOutput |ssc:Boolean |output
+|StringInput |ssc:String |input
+|StringOutput |ssc:String |output
+|===
+
+Modelica connectors of more advanced types are mapped the same way as Modelica component models.
```

Comment:
F2F 2024-06-13: @pmai to provide suggested wording to make the user aware that this very opaque mapping is for now the way to map modelica connectors, but that other, more transparent and detailed mappings might be added in the future, when SSP has better mechanisms, like e.g. terminal support, more specific acausal semantics, etc. without going into details or making any too future looking statements.